src/shared/variables.css

```diff
@@ -25,4 +25,5 @@
 
   /* Borders */
   --rvr-border-radius: 4px;
+  --rvr-tile-border-radius: 6px;
```

Comment:
Separate naming thing:
I think I would label this border radius with "dialog" instead of "tile"

There's no tight definition, but I found the breakdown on this page helpful. https://ux.stackexchange.com/questions/78798/whats-the-difference-between-cards-panels-and-tiles

The article suggests that "tile" is mostly associated with those square in Microsoft's Metro UI, and those aren't like modals at all.

I don't think modals quite fit into any of those categories, but I can imagine we'd end up with a few types of dialog box that should share the same border radius.